src/classes/DML.cls

```diff
@@ -0,0 +1,84 @@
+/*************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.             *
+* See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
+*************************************************************************************************/
+public abstract class DML extends NebulaCore implements IDML {
```

Comment:
This class seems to have grown very large and is becoming difficult to maintain. Please consider refactoring to reduce the number of methods.